pkg/types/openstack/platform.go

```diff
@@ -43,4 +43,13 @@ type Platform struct {
 	// for cluster nodes or an existing Glance image name.
 	// +optional
 	ClusterOSImage string `json:"clusterOSImage,omitempty"`
+
+	// AdditionalSecurityGroups contains names of additional security groups where we want
+	// to add workers
+	// +optional
+	AdditionalWorkerSecurityGroups []string `json:"additionalWorkerSecurityGroups,omitempty"`
+
+	// AdditionalWorkerSubnets contains names of additional subnets where we want to add workers
+	// +optional
+	AdditionalWorkerSubnets []string `json:"additionalWorkerNetworks,omitempty"`
```

Comment:
why is this a platform level option and not machinepool level setting?

Reply:
yep, agree as well!